releasenotes/notes/moved-error-constants-97c7278a381b56d0.yaml

```diff
@@ -0,0 +1,11 @@
+---
+upgrade:
+  - |
+     Instead of using error constants from ``ddtrace.ext.errors``. Use constants from ``ddtrace.constants`` module. 
+     For Example:: ``ddtrace.ext.errors.ERROR_MSG`` -> ``ddtrace.constants.ERROR_MSG``
+      
+deprecations:
+  - |
+    Moved ``ddtrace.ext.errors`` constants into ``ddtrace.constants`` module. 
+    Function ``get_traceback()`` in ddtrace.ext.errors is now deprecated and will be removed.
```

Comment:
Probably worth saying when it will be removed here (1.0)

Reply:
Does something like this work for the deprecation notes?
```
deprecations:
  - |
    Moved ``ddtrace.ext.errors`` constants into ``ddtrace.constants`` module & will be removed in v1.0. 
    Shorthand error constants in ``ddtrace.ext.errors`` will be removed in v1.0.
    Function ``get_traceback()`` in ddtrace.ext.errors is now deprecated and will be removed v1.0.
```

Reply:
👍 I'm sure that we haven't been consistent in mentioning _when_ things will  be deprecated but we can go back to the older release notes and update them as we go... consistency is hard 😕